guides/common/modules/proc_recovering-corrupted-content-on-a-smart-proxy.adoc

```diff
@@ -0,0 +1,32 @@
+[id="Recovering_Corrupted_Content_on_{smart-proxy-context}_{context}"]
+= Recovering corrupted content on {SmartProxy}
+
+In case of content corruption on a {SmartProxy}, you can recover it by using the `verify-checksum` command in Hammer CLI.
```

Comment:
I'm wondering about one thing - how does the user know that they are dealing with content that had been corrupted? Is it clearly indicated to them, or do they have to dig for this information? Perhaps a SME might answer these Qs?

Reply:
@sjha4 Can you elaborate?

Reply:
There are several causes and effects for corrupted data in /var/lib/pulp on the FS in general. I found a nice KB article with some diagnostic steps dealing with identifying such data: https://access.redhat.com/solutions/6961905

In general, if the client is unable to consume content from a published repo that it is subscribed to, we can use repair to diagnose any issues of data corruption. It does a checksum match on the data and identifies if there are any corrupted/missing units and attempts to redownload them from the source